packages/connect-ui/src/components/modal/modal.sass

```diff
@@ -0,0 +1,252 @@
+.modal-container {
```

Comment:
File should be `.scss` right? .sass would be sans-semicolons

Reply:
It compiles, but yes. I personally prefer SASS so I'll likely just strip the semis. Long term, I think you and Thomas have better ideas around CSS-in-JS solutions - we might even be able to share the styled-system-esque solution that Thomas is working on.

Reply:
Yes, I think almost certainly we can use `@stacks/ui-core` and `@stacks/ui-theme` here :) still WIP tho